OOPBASICSPART1TASK6/src/com/gyk/java/oop/basics/Family.java

```diff
@@ -0,0 +1,47 @@
+package com.gyk.java.oop.basics;
+
+import java.util.Arrays;
+
+public class Family {
+    private Person[] persons;
+    private Vehicle vehicle;
+    private TravelDestination travelDestination;
+
+    public Family() {

```

Comment:
If you don't need to create object without parameters then it is not required to have an empty constructor.

Reply:
So, I don't have to think about possible wishes in the future?

Reply:
thank you!